### PR TITLE
Update overview command and DG

### LINKED ISSUE #61: Update overview
add more descriptions

### PR DESCRIPTION
**Pull request by ryan1604**
Fixes #61

### CODE REVIEW

File: src/main/java/seedu/financialplanner/commands/OverviewCommand.java [hunk 1]
```diff
@@ -20,44 +24,74 @@ public OverviewCommand(RawCommand rawCommand) {
...
+
+    private static String getReminders() {
+        ReminderList reminderList = ReminderList.INSTANCE;
+        if (reminderList.list.isEmpty()) {
+            return "No reminders added yet.";
+        }
+        StringBuilder reminders = new StringBuilder();
+        int count = 1;
+        for (Reminder reminder : reminderList.list) {
+            reminders.append(count).append(". ").append(reminder.toString()).append("\n");
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/commands/OverviewCommand.java:92]: is there a neater way of writing this, maybe split into steps

File: src/main/java/seedu/financialplanner/utils/Ui.java [hunk 2]
```diff
@@ -134,4 +135,40 @@ public void printResetBudget() {
...
+            showMessage("Budget has not been set yet.");
+            break;
+        case "reset":
+            showMessage("Budget has not been spent yet.");
+            break;
+        case "view":
+            showMessage("There is no existing budget.");
+            break;
+        default:
+            showMessage("Unknown command");
```
Comment (NeoMinWei) [inline src/main/java/seedu/financialplanner/utils/Ui.java:175]: can logging be added here if default case is not meant to be reached

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: src/main/java/seedu/financialplanner/utils/Ui.java (hunk 2 above)
Comment (ryan1604) [reply src/main/java/seedu/financialplanner/utils/Ui.java:175]: thanks for reminding i missed that

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -63,7 +64,7 @@ Output
 `Displaying piechart for expense`
 A message will be shown telling you that the chart is being displayed
 
-![](C:\Users\puach\Documents\Y2S1\CS2113\tp\docs\images\vis\visOutput.png)
+![](images/vis/visOutput.png)
```
Comment (wwweert123) [inline docs/DeveloperGuide.md:67]: Thanks for helping me change the path!

File: src/main/java/seedu/financialplanner/commands/OverviewCommand.java [hunk 4]
```diff
@@ -20,44 +24,74 @@ public OverviewCommand(RawCommand rawCommand) {
...
     @Override
     public void execute() throws Exception {
-        CashflowList list = CashflowList.getInstance();
-        Ui.getInstance().showMessage("Here is an overview of your financials:\n" +
-                "Total balance: " + formatDoubleToString(Cashflow.getBalance()) + "\n" +
-                "Highest income: " + getHighestIncome(list) + "\n" +
-                "Highest expense: " + getHighestExpense(list) + "\n" +
-                "Remaining budget for the month: " + getBudgetDesc());
-        //todo: indicate description of income/expense
+        Ui.getInstance().printOverview(getBalance(), getHighestIncome(), getHighestExpense(),
```
Comment (wwweert123) [inline src/main/java/seedu/financialplanner/commands/OverviewCommand.java]: Wow, this line look very long. Maybe can separate out and store values in variables for readability

File: docs/diagrams/Budget.puml [hunk 5]
```diff
@@ -10,7 +10,7 @@ return ui
```
Comment (wwweert123) [inline docs/diagrams/Budget.puml:9]: For budget sequence diagram, you might want to add : for objects of classes or add <<class>> if you are interacting with class static methods

**Review by wwweert123 — APPROVED** [review]
LGTM!

File: src/main/java/seedu/financialplanner/utils/Ui.java (hunk 2 above)
Comment (ryan1604) [reply src/main/java/seedu/financialplanner/utils/Ui.java:175]: added

File: src/main/java/seedu/financialplanner/commands/OverviewCommand.java (hunk 4 above)
Comment (ryan1604) [reply src/main/java/seedu/financialplanner/commands/OverviewCommand.java]: changed!

File: docs/diagrams/Budget.puml (hunk 5 above)
Comment (ryan1604) [reply docs/diagrams/Budget.puml:9]: fixed

**Pull request merged**